packages/app/src/cli/commands/app/dev.ts

```diff
@@ -82,7 +85,7 @@ export default class Dev extends Command {
       subscriptionProductUrl: flags['subscription-product-url'],
       checkoutCartUrl: flags['checkout-cart-url'],
       tunnelUrl: flags['tunnel-url'],
-      noTunnel: flags['no-tunnel'],
+      tunnel: flags.tunnel || hasUIExtensions,
```

Comment:
Do we want to move this into the service and test that the right `tunnel` value is used based on the number of extensions?